content/staff-wellbeing/staff-wellbeing-statement.md

```diff
@@ -0,0 +1,133 @@
+
+**Contact details**: Email the school office at <reception@stpetershuntingdon.org> 
+
+{/inset-text}
+
+<div class="govuk-grid-row dfe-width-container">
+  <div class="govuk-grid-column-full">
+    <div class="info-box">
+      <div class="info-box__corner">
+        <img src="/assets/images/bullseye.svg" alt="Bullseye icon">
+      </div>
+      <h2 class="govuk-heading-m">
+        Impact and outcomes
+      </h2>
+      <p>
+        Our united approach creates: 
+        <ul>
+          <li>
+            a sense of belonging
+          </li>
+          <li> 
+            a culture that is based upon our core wellbeing principles 
+          </li>
+          <li>
+            an environment that recognises skills and encourages personal development
+          </li>
+          <li>
+            an environment whereby employees feel they are listened to 
+          </li>
+      </p>
+    </div>
+  </div>
+</div>
+
+## Background from HR Manager Rachel Boyall
+
+{inset-text}
+
+The staff wellbeing statement outlined below was developed by our wellbeing focus group. In addition to the statement, the focus group: 
+
+* offers support to staff 
+
+* seeks feedback on how staff are feeling 
+
+* shares resources 
+
+* looks for ways to boost morale 
+
+* uses staff absence to measure wellbeing  
+
+{/inset-text}
+
+## Our principles 
+
+These principles are the cornerstone of our approach to wellbeing at St Peter’s: 
+
+* because we care 
+
+* because everyone matters 
+
+* together we can make a difference 
+
+* this is worth it 
+
+* we are worth it  
+
+* you are worth it 
+
+## Staff wellbeing statement 
+
+St Peter’s recognises its responsibility for the health, safety and welfare of its employees, and understands that wellbeing and performance are linked. We are committed to fostering a culture of co-operation, trust and mutual respect, where all individuals are treated with dignity, and can work at their optimum level.   
+
+We recognise that work-related stress has a negative impact on employees’ wellbeing and that it can take many forms, so it needs to be carefully monitored and addressed at an organisational level.  
+
+At St Peter’s we: 
+
+* carefully plan and agree work-life balance solutions including flexible working practices where possible and appropriate 
+
+* ensure that the right people are recruited to the right jobs and that a good match is obtained between individuals recruited and job descriptions/specifications 
+
+* manage pressures which may affect employees, including the impact of workload, and anticipate likely problems, taking action to reduce the effects of these pressures where possible 
+
+* pay attention to any changes in employee performance or behaviour and promote sympathetic alertness to employees who show signs of being under stress   
+
+* create opportunities for employees to discuss concerns and enable employees to do so in a supportive environment, through one-to-one meetings and feedback sessions 
```

Comment:
```suggestion
* create opportunities for employees to discuss concerns and enable employees to do so in a supportive environment. This can be through one-to-one meetings and feedback sessions 
```